RealisticPlantSim/Assets/Scripts/Procedural Terrain/DefaultRandomPlacement.cs

```diff
@@ -12,4 +22,9 @@ public override Vector3 randomizePosition(PlantGenerator plantGenerator)
 
```

Comment:
Another way of doing this is:
Vector3 offset = new Vector3(randomXOffset,0,randomZOffset);
 Vector3 position = plantGenerator.transform.position+offset;

Its fine as it is. This is just more readable

Reply:
also yeah I know its just a default impl, so not very important